src/server/scripts/Commands/cs_debug.cpp

```diff
@@ -1597,18 +1523,18 @@ class debug_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleDebugInstanceSpawns(ChatHandler* handler, char const* args)
+    static bool HandleDebugInstanceSpawns(ChatHandler* handler, Variant<uint32, ExactSequence<'e', 'x', 'p', 'l', 'a', 'i', 'n'>> optArg)
     {
         Player const* const player = handler->GetSession()->GetPlayer();
         if (!player)
             return false;
 
         bool explain = false;
         uint32 groupID = 0;
-        if (!stricmp(args, "explain"))
+        if (optArg.holds_alternative<ExactSequence<'e', 'x', 'p', 'l', 'a', 'i', 'n'>>())
```

Comment:
for the record, you can just check uint32 first (or use `.index`) 😛

Reply:
Isn't it nice though? :stuck_out_tongue_closed_eyes: